tox.ini

```diff
@@ -26,43 +27,47 @@ setenv =
     ENABLE_BDIST_EXT_MODULE = {env:ENABLE_BDIST_EXT_MODULE:no}
     OS = {env:OS:}
     COVERAGE_FILE = {env:COVERAGE_FILE:{toxworkdir}/.coverage.{env:OS}.{envname}}
-deps =
-    pytest
-    pytest-cov
-commands=
-    pytest \
+whitelist_externals = poetry
+commands =
+    poetry install
+    poetry run python -m pytest \
         --cov "{envsitepackagesdir}/hera" \
         --cov-config "{toxinidir}/tox.ini"
 
 [testenv:coverage]
 setenv =
     COVERAGE_FILE = {toxworkdir}/.coverage
-skip_install = true
-deps =
-    coverage
+whitelist_externals = poetry
 commands =
-    coverage combine
-    coverage report -m
-    coverage xml -o {toxworkdir}/coverage.xml
-    coverage html -d {toxworkdir}/htmlcov
-depends = python3.7, python3.8, python3.9, python3.10
+    poetry install
+    poetry run coverage combine
+    poetry run coverage report -m
+    poetry run coverage xml -o {toxworkdir}/coverage.xml
+    poetry run coverage html -d {toxworkdir}/htmlcov
+depends = py37, py38, py39, py310
 
 [testenv:lint]
-skipsdist = True
-skip_install = true
-deps =
-    flake8
-allowlist_externals =
-    make
+whitelist_externals = poetry
 commands=
-    make lint
+    poetry install
+    poetry run make lint
+
+[testenv:format]
+whitelist_externals = poetry
+commands =
+    poetry install
+    poetry run make format
 
 [testenv:typecheck]
 skipsdist = True
-allowlist_externals =
-    make
-deps =
-    mypy
-    {[testenv]deps}
+whitelist_externals = poetry
 commands=
-    make typecheck
+    poetry install
+    poetry run make typecheck
+
+[gh-actions]
+python =
+    3.7: py37
+    3.8: py38
+    3.9: py39
+    3.10: py310, lint, typecheck, coverage #, format
```

Comment:
What's the `#, format` comment? 🤔

Reply:
@flaviuvadan These lines are for `tox-gh-actions`,  so I removed these.